website/src/pages/partner-podcast.tsx

```diff
@@ -0,0 +1,151 @@
+        url: '/case-studies/indeed-accelerates-development-velocity',
+        linkText: 'Read the case study',
+    },
+    {
+        name: 'Lyft',
+        logo: '/external-logos/lyft-logo.svg',
+        altTitle: 'Onboard developers',
+        title: 'Decrease time to first commit with self-serve codebase onboarding and knowledge sharing. Learn how Lyft makes every developer autonomous with Sourcegraph.',
+        url: '/case-studies/lyft-monolith-to-microservices',
+        linkText: 'Read the case study',
+    },
+    {
+        name: 'FactSet',
+        logo: '/external-logos/factset-logo.svg',
+        altTitle: 'Promote code reuse',
+        title: 'Find existing libraries for reuse and contribute to a more secure and coherent codebase. See how FactSet ensures consistency across its codebase.',
+        url: '/case-studies/factset-migrates-from-perforce-to-github',
+        linkText: 'Read the case study',
+    },
+]
+
+const PartnerPodcastPage: FunctionComponent<PageProps> = props => {
+    return (
+        <Layout
+            location={props.location}
+            meta={{
+                title: 'Code Intelligence Platform | Sourcegraph',
+                description:
+                    'Address security risks, onboard to a new codebase, identify the root cause of incidents, promote code reuse, improve code health, and more with Sourcegraph.',
+            }}
+        >
+            <div className="bg-gradient-saturn-radial py-md-8 py-6">
+                <section className="container max-w-900">
+                    <h1 className="display-1 font-weight-bold max-w-800">
+                        Accelerate engineering velocity with Sourcegraph
+                    </h1>
+                    <h5 className="py-4 font-weight-normal">
+                        See why over 1.2M engineers use Sourcegraph to build software you rely on
+                    </h5>
+                    <div className="d-flex flex-column flex-md-row pt-0 pt-md-4">
+                        <Link to="/demo" className="btn btn-primary max-w-200">
+                            Request a demo
+                        </Link>
+                        <Link to="/get-started" className="btn btn-outline-primary max-w-150 ml-md-4 mt-4 mt-md-0">
+                            Get started
+                        </Link>
+                    </div>
+                </section>
+            </div>
+
+            <div className="container max-w-900 mt-6">
+                <h1 className="display-3 font-weight-bold">Move fast — even in big code bases</h1>
+                <h5 className="font-weight-normal py-3">Learn how these software companies used Sourcegraph</h5>
+                <div className="d-flex flex-wrap pt-4">
+                    {caseStudyItems.map((study, i) => (
+                        <div key={i} className="col-md-4 mb-6 px-0 partner-pod__case-study-card">
```

Comment:
I think we should do a more progressive stacking here as well. Try this: `col-sm-6 col-md-4`.